lib/ASRouter.jsm

```diff
@@ -268,19 +273,28 @@ class _ASRouter {
     this.messageChannel.sendAsyncMessage(OUTGOING_MESSAGE_NAME, {type: "ADMIN_SET_STATE", data: state});
   }
 
-  async _findMessage(messages, target, data = {}) {
-    let message;
+  _handleTargetingError(type, error, message) {
+    Cu.reportError(error);
+    if (this.dispatchToAS) {
+      this.dispatchToAS({
```

Comment:
should this (or is this) routed to the ASRouter ping center? 

https://github.com/mozilla/activity-stream/blob/868db6f54c2ac36a35fee0ad5880d0a5bbdd6d2f/lib/TelemetryFeed.jsm#L137-L141

Reply:
@k88hudson any reason for not using the `ASRouterUserEvent` defined in Actions.jsm?